packages/react-button/src/components/Button/useButtonBehavior.ts

```diff
@@ -0,0 +1,31 @@
+import { buttonBehavior } from '@fluentui/accessibility';
+import { getCode } from '@fluentui/keyboard-key';
+import { ButtonProps, ButtonState } from './Button.types';
+
+export const useButtonBehavior = (props: ButtonProps): Partial<ButtonState> => {
+  const { as, disabled, loading, onClick, onKeyDown } = props;
+
+  const { attributes, keyActions } = buttonBehavior({ as: as as string, disabled, loading });
+
+  let _onKeyDown = onKeyDown;
+
+  const keyCombinations = keyActions?.root?.performClick?.keyCombinations;
+  if (keyCombinations) {
+    _onKeyDown = (ev: React.KeyboardEvent<HTMLButtonElement> & React.MouseEvent<HTMLButtonElement>) => {
```

Comment:
```suggestion
    _onKeyDown = (ev: React.KeyboardEvent<HTMLButtonElement>) => {
```

I don't think that it's valid. I suggest to update `ButtonProps` instead to include `KeyboardEvent` as a possible option to `onClick` handler

Reply:
Made the change.